scanpipe/management/commands/__init__.py

```diff
@@ -78,10 +78,9 @@ def validate_pipelines(pipelines):
     """
     Raise an error if one of the `pipelines` is not available.
     """
-    for pipeline_location in pipelines:
-        if not scanpipe_app_config.is_valid(pipeline_location):
-            pipeline_list = [location for location, _ in scanpipe_app_config.pipelines]
+    for pipeline_name in pipelines:
```

Comment:
nit: since `for pipeline_name in pipelines` iterates over names, may be `pipelines` should be really be called `pipeline_names` to distinguish with cases where the list of pipelines is really a list of classes or a dict?
I see `pipelines` used as collection name for things that are very different